serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/utils/apicurio/EnumDefinition.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 Red Hat
```

Comment:
Please check my PR for the correct license header. It applies to all other places.

Reply:
Please address this.

Reply:
Same as above. I will address in the next PR.